scripts/run-bitbucket.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -ue
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ODS_PIPELINE_DIR=${SCRIPT_DIR%/*}
+
+HOST_PORT="7990"
+CONTAINER_NAME="bitbuckettest"
+BITBUCKET_IMAGE_TAG="latest"
+K8S_SECRET_FILE="${ODS_PIPELINE_DIR}/test/testdata/deploy/cd-kind/secret-bitbucket-auth.yml"
+K8S_CONFIGMAP_FILE="${ODS_PIPELINE_DIR}/test/testdata/deploy/cd-kind/configmap-bitbucket.yml"
+
+echo "Run container using image tag ${BITBUCKET_IMAGE_TAG}"
+docker rm -f ${CONTAINER_NAME} || true
+docker run -d --name ${CONTAINER_NAME} -e ELASTICSEARCH_ENABLED=false --net kind -p "${HOST_PORT}:7990" -p 7999:7999 atlassian/bitbucket-server:${BITBUCKET_IMAGE_TAG}
+
+BITBUCKET_URL="http://localhost:${HOST_PORT}"
+echo "Waiting up to 3 minutes for Bitbucket to start ..."
+# https://confluence.atlassian.com/bitbucketserverkb/how-to-monitor-if-bitbucket-server-is-up-and-running-975014635.html
+n=0
+set +e
+until [ $n -ge 18 ]; do
+    health=$(curl -s ${INSECURE} "${SONARQUBE_URL}/status" | jq -r .state)
```

Comment:
@michaelsauter Missing variable declaration of `INSECURE`: https://github.com/opendevstack/ods-pipeline/runs/2711388398?check_suite_focus=true#step:6:64